quickstarts/Jumpstart - Quickstart/dashboards/Jumpstart.json

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
Looks like those validation problems are with the dashboard! You'll just need to sanitize the json file, we have a handy script for that over [here](https://github.com/newrelic/newrelic-quickstarts#-dashboards)

Reply:
Ok, done. I merged it in the main at jcountsNR.